### PR TITLE
Integrate manual assistant

### PR DESCRIPTION
**Pull request by dsmits**
Unfortunately I can't really display the actual input values with the info from the fairstep so I'm just displaying the name.

### CODE REVIEW

**Comment by dsmits** [comment]
Coverage is going down because I didn't write tests. Because it is specifically a manual step I think it will be difficult to create an automated test. Open to suggestions though.

File: fairworkflows/manual_assistant.py [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+# TODO: Remove
+USE_TEST_SERVER = True
+EXAMPLE_STEP_URI = 'http://purl.org/np/RAFszXfE-J3sef_ZX_5LRMM6rHgBt7a1uQH-vZdxfy-RU'
+
+HOST = 'localhost'
+PORT = 8000
+ENCODING = 'UTF-8'
+
+#
+# def get_manual_step(uri: str) -> FairStep:
```
Comment (raar1) [inline fairworkflows/manual_assistant.py]: Is this stuff we can get rid of now?

File: fairworkflows/manual_assistant.py [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+from jinja2 import Environment, PackageLoader, select_autoescape
+
+logging.basicConfig(level=logging.INFO)
+env = Environment(loader=PackageLoader('fairworkflows', 'templates'), autoescape=select_autoescape('html'))
+
+# TODO: Remove
+USE_TEST_SERVER = True
+EXAMPLE_STEP_URI = 'http://purl.org/np/RAFszXfE-J3sef_ZX_5LRMM6rHgBt7a1uQH-vZdxfy-RU'
+
+HOST = 'localhost'
```
Comment (raar1) [inline fairworkflows/manual_assistant.py:16]: Could we rename these env params to e.g. `MANUALTASK_HOST` etc and put them in `fairworkflows/config.py`
I think that would complete the integration.

**Review by raar1 — APPROVED** [review]
Great work. I left two small suggestions below

File: fairworkflows/manual_assistant.py (hunk 1 above)
Comment (dsmits) [reply fairworkflows/manual_assistant.py]: done

File: fairworkflows/manual_assistant.py (hunk 2 above)
Comment (dsmits) [reply fairworkflows/manual_assistant.py]: Added the constants to `config.py`

**Pull request merged**